### PR TITLE
Update known cert dirs for updaters

### PR DESCRIPTION
**Pull request by ChaitanyaKulkarni28**

### CODE REVIEW

File: google_guest_agent/agentcrypto/mtls_mds_linux.go [hunk 1]
```diff
@@ -51,39 +69,41 @@ func (j *CredsJob) writeRootCACert(ctx context.Context, content []byte, outputFi
 
 // writeClientCredentials stores client credentials (certificate and private key).
 func (j *CredsJob) writeClientCredentials(plaintext []byte, outputFile string) error {
+	if err := os.MkdirAll(filepath.Dir(outputFile), 0655); err != nil {
```
Comment (dorileo) [inline google_guest_agent/agentcrypto/mtls_mds_linux.go:72]: the requirement is the same, so for consistency sake add the same comment here: "// The directory should be executable, but the file does not need to be."

**Review by dorileo — COMMENTED** [review]
PR looking good, just minor picky comment.

File: google_guest_agent/agentcrypto/mtls_mds_linux.go (hunk 1 above)
Comment (ChaitanyaKulkarni28) [reply google_guest_agent/agentcrypto/mtls_mds_linux.go:73]: done

**Pull request merged**